pkgs/development/libraries/exiv2/default.nix

```diff
@@ -106,6 +106,10 @@ stdenv.mkDerivation rec {
       rm *
       mv .exiv2 exiv2
     )
+
+    # Tell cmake to look in the default output to find exiv2's .so and .a files
+    substituteInPlace $out/lib/cmake/exiv2/exiv2Config-release.cmake \
```

Comment:
Huh? Should not this file be in `dev` output?

Reply:
Should it? Just the `libexiv2-xmp.a`, right? I could change the derivation to move that to the `dev` output instead, if you think that'd be a better solution. Was trying to be minimally invasive here.

Reply:
I mean the `lib/cmake` directory.

Reply:
Oh, right! Attempting to do that (`substituteInPlace $dev/lib/cmake/...`) gives me an error that `file '/nix/store/...-exiv2-0.27.2-dev/lib/cmake/exiv2/exiv2Config-release.cmake' does not exist`. Seems like that tree is still inside `$out` during `postInstall`? (I'm not entirely sure how multiple output packages work, and skimming the nixpkgs manual isn't clarifying it for me.)

Reply:
Oh, right it is moved during `preFixup` by the setup hook:

https://github.com/NixOS/nixpkgs/blob/bf47b821f9b9d923942dbb2c60fa11b956d28dd9/pkgs/build-support/setup-hooks/multiple-outputs.sh#L4

Reply:
Looks like `lib/cmake` gets moved to the `dev` output during `preFixup` phase, which comes _after_ this `postInstall` script.

So is this OK, or does a change need to be made?

(Cite: https://github.com/NixOS/nixpkgs/blob/20.03/pkgs/build-support/setup-hooks/multiple-outputs.sh#L4)

Reply:
Should be fine. Could you include a link to https://github.com/NixOS/nixpkgs/issues/95232 in the comment, though?